loader/plugin-loader.js

```diff
@@ -36,6 +37,8 @@ module.exports = class PluginLoader {
 
       return plugins
     }, [])
+
+    this.loadedPlugins.push(helpPlugin)
```

Comment:
This needs to happen before the other plugins are loaded so it's always on top of the `!plugins` and `!help` list + gets highest priority in command naming.